apps/edxapp/templates/services/nginx/configs/cms.conf.j2

```diff
@@ -6,10 +7,10 @@ server {
   listen {{ edxapp_nginx_cms_port }};
   server_name localhost;
 
-  {% if activate_http_basic_auth or edxapp_activate_http_basic_auth -%}
+{% if http_basic_auth_enabled and edxapp_nginx_cms_bypass_htaccess_ip_whitelist | length == 0 %}
```

Comment:
Following the same pattern as for `http_basic_auth_enabled`, can you define a `bypass_htaccess` variable to improve readability?

Reply:
:+1: